src/components/Theme/Theme.component.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { ThemeProvider } from 'styled-components';
+
+import { dark } from '../../utils/constants/theme';
+
+const Theme = ({ children }) => <ThemeProvider theme={dark}>{children}</ThemeProvider>;
+
+export default Theme;
```

Comment:
This is a good approximation of how to create a provider, after we learn about Hooks, this can be improved